packages/website/src/docs/toggle-button.mdx

```diff
@@ -25,25 +38,62 @@ component: ToggleButton
 </ToggleButtonGroup>
 ```
 
-## ToggleButtonGroup
+### System props
 
-### Custom props
+<SystemProps props={['margin']} />
 
-| Name       | Type              | Default  | Description                                        |
-| :--------- | :---------------- | :------: | :------------------------------------------------- |
-| `variant`  | 'normal' \|'fade' | `normal` | Variant to display                                 |
-| `value`    | String \| Array   |  `null`  | Value of ToggleButton                              |
-| `onChange` | Function          |  `null`  | Callback when a ToggleButton selection has changed |
+<Heading as="h2" size={4} fontWeight={500} mb={2} mt={6} lineHeight={3}>
+  ToggleButton
```

Comment:
Is there a reason why these 2 are inside the same page? as opposed to say Menu/MenuItem or RadioGroup/Radio, which have separate pages?

Reply:
Well spotted. I thought about that... the answer is, I don't know. I was experimenting with different ways to display them. I think when we document Components with multiple parts on the same page it becomes hard to read and follow. But on the other hard, you can't use a `ToggleButton` without a `ToggleButtonGroup`, whereas with `MenuItem` and `Radio` you could.

Any suggestions?

Reply:
I feel maybe the name correlation is enough for people to make sense that they work together no? So having different page is cool to me. Also we can link from one to the other and be a bit more insistent in the docs that they go together.